src/components/organisms/NoteStorageNavigator.tsx

```diff
@@ -230,6 +231,18 @@ const NoteStorageNavigator = ({ storage }: NoteStorageNavigatorProps) => {
     }
   }, [createNoteByRoute])
 
+  const { toggleShowSearchModal } = useSearchModal()
```

Comment:
@Rokt33r Readded as this fixes keymap changing and initially bound accelerator.
Tried to see if it breaks `Ctrl+P` shortcut in cloud space, seems to work fine (opens its own modal search).

And it works in local space too, but if you can confirm and double-check this?

Reply:
Should be no problems since this component and the webview are never rendered in a same page.